src/components/InputField/InputFieldStyled.js

```diff
@@ -53,53 +56,70 @@ export const StyledInput = styled.input`
   padding-right: ${({ type }) => (type === 'password' ? '48px' : '16px')};
 
   @media screen and (min-width: 768px) {
-    background-color: ${({ $backgroundcolor }) => $backgroundcolor || 'var(--color-grey-4)'};
+    background-color: ${({ $backgroundcolor }) =>
+      $backgroundcolor || 'var(--color-grey-4)'};
   }
 `;
 
 export const StyledTextarea = styled(TextareaAutosize)`
   box-sizing: border-box;
   width: 100%;
   resize: none;
-  font-family: "Roboto", sans-serif;
+  font-family: 'Roboto', sans-serif;
```

Comment:
We already have font-family in the GlobalStyles, I think it`s unnecessary to repeat it here.

Reply:
Removed